srcpkgs/stk/template

```diff
@@ -66,7 +66,6 @@ post_install() {
 
 stk-data_package() {
```

Comment:
Why not kill this data package?

Reply:
I don't know. Maybe we should kill this -data too.

Reply:
No, there're other packages depend on `stk-data`. I'm better to not push on a rush.